packages/core/test-support/helpers/wait-for-element.js

```diff
@@ -9,7 +9,7 @@ import { Promise } from 'rsvp';
 /**
  * Helper to wait for element to be loaded
  *
- * @param {Object} selector - element selector
+ * @param {String} selector - element selector
```

Comment:
Can replace this with: https://github.com/emberjs/ember-test-helpers/blob/master/API.md#waitfor